src/moj/components/date-picker/README.md

```diff
@@ -1,7 +1,7 @@
 # Date picker
 
 - [Guidance](https://design-patterns.service.justice.gov.uk/components/date-picker
-picker)
+  picker)
```

Comment:
Noticed this [extra line was accepted previously](https://github.com/ministryofjustice/moj-frontend/pull/601/files#r1698511472), should it be?